modules/pcd/src/lib/parse-pcd.js

```diff
@@ -66,7 +66,7 @@ function getNormalizedHeader(PCDheader, attributes) {
 function getNormalizedAttributes(attributes) {
   const normalizedAttributes = {
     POSITION: {
-      value: new Float32Array(attributes.position),
+      value: new Float64Array(attributes.position),
```

Comment:
Binary PCD is only 32 bit, but I suppose ASCII PCD could have more precision (haven't checked spec). 

If we want to support that we should move the conversion to the separate branches and document that this module can return different precision depending on mode.

Reply:
https://github.com/visgl/loaders.gl/blob/master/modules/pcd/src/lib/parse-pcd.js#L235